contents/swift/welcome-to-swift.md

```diff
@@ -58,16 +58,14 @@ func yearAlbumReleased(name: String) -> Int? {
 
 And here is how you unwrap the Optional safely:
 
-``` swift
+```swift
 func timeTravel(album: String) {
-    let year = yearAlbumReleased(album) // type of year: Int?
-    if let past = year {
-        // past is a normal (non-optional) Int value
-        // equal to the value stored in year
-        travelTo(past)
-    } else {
-        // year did not hold a value
-        liveInPresent()
+    let year: Int? = yearAlbumReleased(album)
```

Comment:
This makes the type of year clearer but not sure if you want to use/make a note about type inference

Reply:
Ok, I've added this as a separate segment above so that I don't have to introduce a new concept of `type inference` here.